packages/manager/apps/pci-kubernetes/src/pages/detail/nodepools/new/New.page.tsx

```diff
@@ -133,12 +150,15 @@ export default function NewPage(): JSX.Element {
         return 'available';
       })();
 
-      const warn = store.isMonthlyBilling && store.autoScaling.isAutoscale;
+      const warn = Boolean(
+        store.isMonthlyBilling && store.autoScaling?.isAutoscale,
+      );
 
       const isAntiAffinityEnabled =
         !(
           !store.autoScaling?.isAutoscale &&
-          store.autoScaling?.quantity.desired > ANTI_AFFINITY_MAX_NODES
+          store.autoScaling &&
+          store.autoScaling.quantity.desired > ANTI_AFFINITY_MAX_NODES
```

Comment:
difficult to read, this could work maybe: 

```
const isAntiAffinityEnabled = store.autoScaling?.isAutoscale === true &&
  store.autoScaling.quantity.desired <= ANTI_AFFINITY_MAX_NODES;
```